tedana/selection/_utils.py

```diff
@@ -8,6 +8,17 @@
 LGR = logging.getLogger(__name__)
 
 
+def reorder_dataframe(comptable):
```

Comment:
This may be pedantic, and therefore I don't hold this opinion strongly, but because we're also stripping out the semicolons here, might `clean_dataframe` be a more appropriate name?